test/unit/dsl/objects_test.rb

```diff
@@ -78,6 +78,40 @@
     end
 
     it 'constructs a simple resource+argument with method calls' do
+      obj.qualifier = [['resource'], ['hello', 'world']]
```

Comment:
@chris-rock I have pushed up test/integration/matcher_test.rb for the 'be_in' matcher with tests that mirror the cmp matcher tests.
Please take a look.
Thank you